src/Controllers/CrudTraits/CrudIndexTrait.php

```diff
@@ -3,15 +3,45 @@
 namespace Supaapps\Supalara\Controllers\CrudTraits;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Str;
 
 trait CrudIndexTrait
 {
     public function index(Request $request)
     {
         $query = $this->model::query();
+
         if (!is_null($this->searchField) && $request->has('search')) {
             $query->where($this->searchField, 'LIKE', '%' . $request->get('search') . '%');
         }
+
+        // SEARCH BY MULTIPLE FIELDS -----------
+        if ($request->has('search') && is_array($this->searchFields)) {
```

Comment:
`searchFields` is statically defined as array, maybe we need to check count not type